.github/workflows/ci_cd_release.yml

```
@@ -95,7 +113,7 @@ jobs:
     runs-on: ubuntu-latest
     needs: rolling-release
     steps:
-      - uses: ansys/actions/code-style@main
+      - uses: ansys/actions/code-style@feat/check-actions-security-action
```

```suggestion
      - uses: ansys/actions/code-style@main
```
Needs to be reverted before merging